core/src/main/java/com.panhuk.core/Constants.kt

```diff
@@ -1,3 +1,5 @@
 package com.panhuk.core
 
-const val ERROR = "ERROR"
\ No newline at end of file
+const val ERROR = "ERROR"
+const val RIGHT_ANSWERS = "right answers"
+const val TOTAL_ANSWERS = "total answers"
```

Comment:
If you using it as a bundle key better to make the key name unique and distinctive from any other result to avoid confusion when need to debug all that.
```suggestion
const val CORRECT_ANSWERS = "bundle.correct-answers_key"
const val TOTAL_ANSWERS = "bundle.total-answers_key"
```

Reply:
fully agree. Thanks